study.md

```diff
@@ -67,5 +69,6 @@ Starting with a client making a GET request to a particular URL, describe how
 the parts of Rails interact to produce and send a response.
 
 ```md
-<!-- your answer here -->
+A get request starts at the web browser and a router will check a router file to determine which controller it should use to handle the get request. The controller will then get the requested data from a model, then the controller will return a response with JSon data from the model with a view component that displays the data in an HTML file that is assembled by controller.
```

Comment:
We won't be using Rails's views.  The controllers don't necessarily have to return JSON.  It can return binary data, XML, nothing at all, or anything else.